uber/site_sections/accounts.py

```diff
@@ -99,6 +99,54 @@ def bulk(self, session, department_id=None, **params):
             'attendees': attendees
         }
 
+    def access_groups(self, session, message='', **params):
+        if 'id' in params and params['id']:
+            access_group = session.access_group(params['id'])
+        else:
+            access_group = AccessGroup()
+
+        if cherrypy.request.method == "POST":
+            access_group.name = params['name']
+
+            for key in params:
```

Comment:
Not the most graceful chunk of code, but with the crunch for launching Super I'm prioritizing functionality and speed. This goes through all the selected options on the page and sorts them into the `access` and `read_only_access` columns. If an access or read_only_access is set to 0, it either ignores it or deletes it from the column.